text/0000-Project-Goals-2024h2.md

```diff
@@ -0,0 +1,508 @@
+| ↳ [Production use of annotate-snippets](https://rust-lang.github.io/rust-project-goals/2024h2/annotate-snippets.html#ownership-and-team-asks)                                                 | [Esteban Kuber][], [Scott Schafer][] | [Esteban Kuber][] will be the reviewer |
+
+### crates-io team
+| Goal                                                                                  | Owner      | Notes |
+| ---                                                                                   | ---        | --- |
+| *Approve RFC*                                                                         |            |  |
+| ↳ [Yank crates with a reason](https://rust-lang.github.io/rust-project-goals/2024h2/yank-crates-with-a-reason.html#ownership-and-team-asks)   | [二手掉包工程师][] |  |
+| *Standard reviews*                                                                    |            |  |
+| ↳ [Yank crates with a reason](https://rust-lang.github.io/rust-project-goals/2024h2/yank-crates-with-a-reason.html#ownership-and-team-asks)   | [二手掉包工程师][] |  |
+| *Try it out in crates.io*                                                             |            |  |
+| ↳ [Yank crates with a reason](https://rust-lang.github.io/rust-project-goals/2024h2/yank-crates-with-a-reason.html#ownership-and-team-asks)   | [二手掉包工程师][] |  |
+
+### lang team
+| Goal                                                                                                                                  | Owner            | Notes               |
+| ---                                                                                                                                   | ---              | ---                 |
+| *Design meeting*                                                                                                                      |                  |                     |
+| ↳ [Async closures](https://rust-lang.github.io/rust-project-goals/2024h2/async.html#ownership-and-team-asks)                                                                                  | [Michael Goulet][] | 2 meetings expected |
+| ↳ [Trait for async iteration](https://rust-lang.github.io/rust-project-goals/2024h2/async.html#ownership-and-team-asks)                                                                       | [Eric Holk][]           | 2 meetings expected |
+| ↳ [Async drop experiments](https://rust-lang.github.io/rust-project-goals/2024h2/async.html#ownership-and-team-asks)                                                                          | [Vadim Petrochenkov][]    | 2 meetings expected |
+| ↳ [Ergonomic ref-counting](https://rust-lang.github.io/rust-project-goals/2024h2/ergonomic-rc.html#ownership-and-team-asks)                                                                   | [Jonathan Kelley][]      | 2 meetings expected |
```

Comment:
If we actually used all of these design meetings it would take up around 2/3 of our capacity for the remainder of the year, by my estimate. I think I'm okay with this given that

- we likely won't use all of them
- even if we do, these are probably the most important things to be spending them on
- we might get more slots if we continue with only doing a planning meeting every two months

That said, I would like to do a retrospective periodically and at the end of the year to check if we feel like we spent our effort on the right things.

Reply:
Yes, I felt the same. But it's good to have a check on the capacity.